app/Models/Configs.php

```diff
@@ -221,6 +221,7 @@ public static function getValueAsBool(string $key): bool
 	 */
 	public static function getValueAsEnum(string $key, string $type): \BackedEnum|null
 	{
+		// @phpstan-ignore-next-line
```

Comment:
Due to typehinting on getValueAsEnum, it is assumed that `$type` will be a BackedEnum, this is not necessarily the case... in practice.

Reply:
Is it worth putting this comment in the file instead?

Reply:
Fixed [7f6ffca](https://github.com/LycheeOrg/Lychee/pull/1764/commits/7f6ffcaf4f3eb412af1e97be894ae779df82dd85).